OpenRA.Mods.Common/Activities/FindAndDeliverResources.cs

```diff
@@ -24,6 +24,7 @@ public class FindAndDeliverResources : Activity
 		readonly HarvesterInfo harvInfo;
 		readonly Mobile mobile;
 		readonly ResourceClaimLayer claimLayer;
+		readonly LinkClientManager linkClient;
```

Comment:
It is really hard to understand what this API is doing by its name. Maybe
```suggestion
		readonly DropoffCoordinator dropoffCoordinator;
```

Reply:
This class does a load of things

1. Manages link orders
1. Acts as a mediator between link clients and hosts
1. Acts as a way to look for potential hosts by outside scripts
1. Contains `LastReservedHost`